index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a70cf7601d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+
+  <body>
+    <section>
+      <header class="header">
+        <nav class="nav">
+          <a href="">Welcome</a>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <div class="hero-heading">
+        <h1>Hey there, I am Dennis.</h1>
+        <h1 id="title">I'm a software Developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-icons">
```

Comment:
- [ ] well done on including this section with all it's icons and having them as correct links. Please also make sure to use a `ul` tag here instead of a `div`. a `ul` which is an unorder-list is more semantically correct here and we always want to have HTML that is as semantically meaningful as possible 🤗.
- [ ] Make sure also that each of the children of the `ul` are `li`s. so you can have the `ul` container and `li` children and each `li` has an `a` link that is an icon 👍

Reply:
ok. This has been fixed.